portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -31,19 +38,32 @@
 
```

Comment:
Somehow I can't select the line in Github, but this comment is meant for line 37: `private final ArrayList<String> comments = new ArrayList<>();`

Do you still need this ArrayList if you are storing comments in the datastore?

Reply:
nope, forgot to remove

Reply:
Actually I didn't see a commit deleting this line though?